docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -24,54 +24,50 @@
 
 @endsphinxdirective
 
-## Introduction
-
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
 
-Using Model Optimizer tool assumes you already have a deep learning model trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX* format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
+When using Model Optimizer, you are assumed to already have a deep learning model that is trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
 
-> **NOTE**: Model Optimizer does not infer models. Model Optimizer is an offline tool that converts a model into IR and optimizes before the inference takes place.
+> **NOTE**: Model Optimizer does not infer models. It is an offline tool that converts a model into IR and optimizes it before the inference takes place.
 
 The scheme below illustrates the typical workflow for deploying a trained deep learning model:
```

Comment:
```suggestion
The figure below illustrates the typical workflow for deploying a trained deep learning model:
```

Reply:
Alright! I think we should add a new point to our alignment guide :) Let's not use scheme! :D